tests/workbenches/conftest.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+from __future__ import annotations
+
+from typing import Generator
+
+from kubernetes.dynamic import DynamicClient
+
+from ocp_resources.namespace import Namespace
+from ocp_resources.persistent_volume_claim import PersistentVolumeClaim
+
+import pytest
+
+from utilities import constants
+from utilities.infra import create_ns
+
+
+@pytest.fixture(scope="class")
+def users_namespace(
```

Comment:
you can use `unprivileged_model_namespace`  https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/model_serving/model_server/authentication/conftest.py#L334 fixture; move it to upper conftest.
you can also add to `create_ns` an new bool arg - `add_dashboard_label` and apply the logic there

Reply:
such refactoring in my very first PR is a bit too ambitious for me, I'd rather not do it <img src="https://github.com/user-attachments/assets/f0897767-7017-43a3-b729-0c0d0e0a6400" alt="please" width="40"/>

Reply:
please see https://github.com/opendatahub-io/opendatahub-tests/pull/184

Reply:
seen and adopted, thanks